.gitignore

```diff
@@ -78,6 +78,19 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
+#
+firebase-debug.log*
+firebase-debug.*.log*
+
+# Firebase cache
+.firebase/
+
+# Firebase config
+
+# Uncomment this if you'd like others to create their own Firebase project.
+# For a team working on the same Firebase project(s), it is recommended to leave
+# it commented so all members can deploy to the same project(s) in .firebaserc.
+.firebaserc
```

Comment:
Right now it's anyways not pushed?

Reply:
didn't get it, the config will be created on adding firebase project by running ```firebase add --use```

Reply:
so you haven't yet added any firebase project with it, right?

Reply:
yes